manage.py

```diff
@@ -3,6 +3,10 @@
 import sys
 
 if __name__ == "__main__":
+    # Add argument needed (and easily forgotten) when using Vagrant.
+    if 'runserver' in sys.argv and len(sys.argv) == 2:
```

Comment:
You're welcome. I'm glad to help.

Could you share your reasoning on why you think this is bad practice?

I'll share my perspective so we can dialog.

1. `manage.py` is generated by django, but it's *user code* and part of the project.
2. this change does not modify how `manage.py` works. It only improves a special case.
3. `runserver` is *only* used during development which is what the patch tries to facilitate.
4. In earlier Django versions `manage.py` was *the entry point* for the project, but since version 1.4 it became easier to [load django on any python module](https://docs.djangoproject.com/en/1.7/releases/1.4/#django-core-management-setup-environ).

I see 2 other alternatives to this:

1. Implementing a new management command to either override `runserver` or add a new command. Both seems overkill to me.
2. Add a `runserver.sh` that calls `manage.py runserver 0.0.0.0:8000` and reference it on the README. It's not too bad, but diverges the common `manage.py` usage and assumes people won't forget to reference the README.